packages/cli/src/commands/contract/test.ts

```diff
@@ -25,13 +25,13 @@ export class CompileContract extends Command {
   static args = {
     contractName: Args.string({
       name: "contractName",
-      required: true,
+      default: "",
       description: "Name of the contract to test",
     }),
   };
 
   async run(): Promise<void> {
-    const { args, flags } = await this.parse(CompileContract);
+    const { args, flags } = await this.parse(TestContract);
 
     if (args.contractName == "" && !flags.all) {
```

Comment:
generally, double equals comparison should be avoided in favour of `===` because of possible false positives. (will catch all falsy values).

Reply:
Even though oclif Arg default is set to empty string "", args.contractName is undefined. I'm not sure why this behaviour